app/controllers/application_controller.rb

```diff
@@ -62,4 +62,8 @@ def render_unauthorized
     headers['WWW-Authenticate'] = 'Token realm="Application"'
     render json: 'Not Authorized', status: 401
   end
+
+  rescue_from ActionController::ParameterMissing do
+    render json: { error: 'parameter missing' }, status: :bad_request
+  end
```

Comment:
I think this is OK for now, I know @saneshark has some ideas on how we should be handling exceptions but that can be address separately. For now, can this be moved to be closer to the other `rescue_from` https://github.com/department-of-veterans-affairs/vets-api/pull/70/files#diff-55c5b7aecfb519d0e4880eaf2788eb6eR18 just for awareness for other devs
I'm not entirely sure how `rescue_from` works when there are multiple handlers defined so maybe